publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/grant/CuratorPermissionStrategy.java

```diff
@@ -52,42 +46,22 @@ private boolean userIsFromSameInstitutionAsPublication() {
             return false;
         }
 
-        var resourceOwnerTopLevel = getTopLevelOrgUri(uriRetriever,
```

Comment:
removed getTopLevelOrgUri lookup here. had a look at resourceOwner on both a imported and a newly created resource and it looks like its top level every time